### PR TITLE
Init

### PR DESCRIPTION
**Pull request by adamklepacz**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    links:
+    - server
+  server:
+    restart: always
+    build: 
+      context: .
+      dockerfile: src/server/Dockerfile
+    ports:
+    - "8080:8080"
+    # environment:
```
Comment (marekrozmus) [inline docker-compose.yml]: please remove comments

File: README.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
+# simple-react-full-stack
```
Comment (marekrozmus) [inline README.md]: please remove not relevant information

File: src/client/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Header(props) {
```
Comment (marekrozmus) [inline src/client/components/Header.js]: please use arrow syntax:
```
const Header = props => {
...
}
```

File: src/server/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+const app = express();
+
+app.use(express.static('dist'));
+
+// Index endpoint
+app.get('/', (req, resp) => resp.send('Hello World'));
+
+// Example endpoint
+app.get('/api/getUsername', (req, res) =>
```
Comment (marekrozmus) [inline src/server/index.js]: Please use kebab case for apis or path /api/user/name. Please do not use camel case for api endpoints.

File: src/client/App.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import Header from './components/Header';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { username: null };
+  }
+
+  componentDidMount() {
+    fetch('/api/getUsername')
```
Comment (jakub-bogacz) [inline src/client/App.js]: Please change that also, according to the server endpoint name

File: package.json [hunk 6]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "react-full-stack",
```
Comment (marekrozmus) [inline package.json]: name is incorrect

File: package.json [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  "version": "1.0.0",
+  "description": "React, Node.js, Express, MongoDB and Webpack.",
```
Comment (marekrozmus) [inline package.json]: description is incorrect

File: package.json [hunk 8]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "react-full-stack",
+  "version": "1.0.0",
```
Comment (marekrozmus) [inline package.json]: please change to 0.0.1 - it is not ready yet :)

File: public/index.html [hunk 9]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```
Comment (marekrozmus) [inline public/index.html]: please change indentation to two spaces

**Pull request merged**